src/dcommands/help.js

```diff
@@ -31,31 +31,34 @@ class HelpCommand extends BaseCommand {
                     return stringify(this);
                 }
             };
-            if (CommandManager.commandList.hasOwnProperty(words[1]) && !CommandManager.commandList[words[1]].hidden
-                && (!CommandManager.commandList[words[1]].onlyOn || CommandManager.commandList[words[1]].onlyOn === msg.guild.id)) {
-                let instance = CommandManager.built[CommandManager.commandList[words[1]].name];
-                let definition = CommandManager.commandList[words[1]];
-                embed.title = `Help for ${definition.name}`;
-                embed.url = `https://blargbot.xyz/commands#${definition.name}`;
-                embed.description = `**__Usage__**:\`${definition.usage}\`\n${definition.info}`;
-                embed.color = this.getColor(instance.category);
-                if (instance.aliases && instance.aliases.length > 0)
-                    embed.fields.push({
-                        name: '**Aliases**',
-                        value: instance.aliases.join(', ')
-                    });
-                if (instance.flags && instance.flags.length > 0)
-                    embed.fields.push({
-                        name: '**Flags**',
-                        value: instance.flags.map(flag => `\`-${flag.flag}\` or \`--${flag.word}\` - ${flag.desc}`).join('\n')
-                    });
-            } else {
-                let helpText = await bu.ccommand.gethelp(msg.guild.id, words[1]);
-                if (helpText) {
+            let customCommand = await bu.ccommand.get(msg.guild.id, words[1]);
+            if(customCommand) {
+                let helpText = customCommand.help;
+                if(helpText) {
                     embed.color = this.getColor('CUSTOM');
                     embed.title = `Help for ${words[1].toLowerCase()} (Custom Command)`;
                     embed.description = helpText;
                 }
+            } else {
+                if (CommandManager.commandList.hasOwnProperty(words[1]) && !CommandManager.commandList[words[1]].hidden
+                && (!CommandManager.commandList[words[1]].onlyOn || CommandManager.commandList[words[1]].onlyOn === msg.guild.id)) {
+                    let instance = CommandManager.built[CommandManager.commandList[words[1]].name];
+                    let definition = CommandManager.commandList[words[1]];
```

Comment:
Can be a `const` as the `definition` variable is not overwritten somewhere else.

Reply:
Same for my reply above